ramls/userEvent.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
This defines a different structure for the event message than the [original pattern](https://github.com/folio-org/mod-inventory-storage/blob/f40481bf2cbf5a1385141e5169e633b8c15a83d6/src/main/java/org/folio/services/domainevent/DomainEvent.java) established in mod-inventory-storage when the Falcon team did the inventory search work.

What is the reasoning for using a different structure?

Reply:
> First and foremost, the action type is missing there. We plan to add support for DELETE as well. 

The action type is not missing in that structure. The `type` property represents that type. `DELETE` is one of the types used - https://github.com/folio-org/mod-inventory-storage/blob/f40481bf2cbf5a1385141e5169e633b8c15a83d6/src/main/java/org/folio/services/domainevent/DomainEvent.java#L82